File: C/Iniciante/1099somadeimparesconsecutivos2.c
#include <stdio.h>
int main() {
    int n,i,j,x,y,soma,mx,my;
    scanf("%d",&n);
    for(i=1;i<=n;i++){
        soma=0;
        scanf("%d %d",&x,&y);
        if(x<y){
            mx=x;
            my=y;
        }else{
            mx=y;
            my=x;
        }
        for (j = mx+1; j < my; j++) {
            if((j%2)!=0){
                soma+=j;
            }
        }
        printf("%d\n",soma);
    }
    return 0;
}

File: Python/iniciante/1099somadeimparesconsecutivos2.py
n = int(input())
while n:
    soma = 0
    x, y = input().split(" ")
    x = int(x)
    y = int(y)
    if x < y:
        mx = x
        my = y
    else:
        mx = y
        my = x
    for i in range(mx+1, my):
        if i % 2 != 0:
            soma += i
    print("%d" % soma)
    n -= 1
